Defensive.html

```diff
@@ -50,4 +50,4 @@ <h1 style="float:left;">Defensive Drills: </h1>
   <iframe width="500" height="315" src="https://www.youtube.com/embed/RaJMWfUJmaU" frameborder="0" allow="autoplay; encrypted-media"
     allowfullscreen></iframe>
   <iframe width="500" height="315" src="https://www.youtube.com/embed/NrgujFX3wa0" frameborder="0" allow="autoplay; encrypted-media"
-    allowfullscreen></iframe>
\ No newline at end of file
+    allowfullscreen></iframe>
```

Comment:
Is this change necessary? Can you please revert it?

Reply:
@TheSalarKhan

Reply:
Yes its necessary...